subsys-dsp.typ

```diff
@@ -2,6 +2,42 @@
 
 The Data Streaming Protocol (DSP) facilitates the streaming of metric data from probes and the issuance of commands to probes. It is optimized for minimal connection setup time and robustness to packet loss and signal degradation.
 
+== Architecture
+
+The Data Streaming Protocol (DSP) is a specialized communication framework designed to facilitate real-time data streaming from space probes across vast distances. It comprises a dual-module setup with client-side components on the probes and server-side components at Earth stations. 
```

Comment:
Should have caught this in the previous review, but the DSP needs to be able to issue commands to probes as well.